clayui.com/content/docs/components/forms/form.mdx

```diff
@@ -62,4 +62,26 @@ Use `ClayForm.Text` for creating a caption on Form Groups.
 
 ## API
 
+### Form
+
 <div>[APITable "clay-form/src/Form.tsx"]</div>
+
+### Form.FeedbackGroup
+
+<div>[APITable "clay-form/src/Form.tsx(FeedbackGroup)"]</div>
```

Comment:
It seems a bit odd to see the component specification like this `"clay-form/src/Form.tsx(FeedbackGroup)"`, maybe because it looks like I see a function being called 😅. I think using `#` would be better, it would be almost the same as `svg:use` API for example.

```
<svg class="lexicon-icon lexicon-icon-drop" focusable="false" role="presentation">
	<use href="/images/icons/icons.svg#drop" />
</svg>
```

Reply:
Oh thats a good point. Ill change to `#`